client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -219,6 +221,12 @@ module.exports = React.createClass( {
 			activeTabLoading = this.props.activeTabVisitsList.isLoading() && this.props.visitsList.isLoading(),
 			classes;
 
+		analytics.tracks.recordEvent( 'calypso_stats_chart_tab_view', {
```

Comment:
My logic was, that by using this event we can start gathering data not only for the immidiate tests, but also for gathering data for other tabs,
But yeah, it now darned on me that bumping stat in render can be bad.
I will do as you say.